chain/client/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2018"
 
 [dependencies]
 ansi_term = "0.12"
-actix = "0.11.0-beta.2"
+actix = "=0.11.0-beta.2"
```

Comment:
:+1: it's good idea to pin pre-releases which are used in production.

Unrelated to this PR, but I've noticed that we need to update this in a lot of Cargo.tomls. It seems that some deps shouldn't be there? For example, the one in chain/chunks seems to be removable.